docs/v1/P4Runtime-Spec.mdk

```diff
@@ -5329,30 +5329,30 @@ message PacketMetadata {
   the section on [Stream Error Reporting](#sec-stream-error-reporting) for more
   information on `error`.
 
-## Master Arbitration Update
+## Client Arbitration Update
 
-P4Runtime's master arbitration mechanism ensures that only the current master
+P4Runtime's client arbitration mechanism ensures that only the current primary
 can modify state on the switch, and that the `election_id` is monotonically
 increasing. For example, the switch must finish all previous write operations
-before changing masters, and must only accept write requests from the current
-master.
+before changing primarys, and must only accept write requests from the current
```

Comment:
s/primarys/primaries

or "before selecting a different primary"

Reply:
I like the latter better.